src/shadowbox/model/access_key.ts

```diff
@@ -37,6 +45,10 @@ export interface AccessKey {
   metricsId: AccessKeyMetricsId;
   // Parameters to access the proxy
   proxyParams: ProxyParams;
+  // Admin-controlled, data transfer quota for this access key. Unlimited if unset.
+  quota?: AccessKeyQuota;
```

Comment:
How do we represent "default" quota?

Reply:
The default quota should be a server property, persisted in the server config. We agreed that default quotas would be applied to new access keys. I'm thinking we could add an `isDefault` property to `AccessKeyQuota`. We would update the quotas where this property is true when setting the default quota.

Stepping back on previous assumptions, the default quota behavior may benefit from UX research. I plan to implement default quotas in a follow-up PR.

Reply:
We need to be able to differentiate overridden quota from default quota, even if they are the same. This differentiation will be needed for serialization, but probably also the UI.

I suggest we use the absence of the field to mean default quota, which, by default, is unlimited access. You just need to update the comment.